boostedblob/azure_auth.py

```diff
@@ -51,20 +51,13 @@ def load_credentials() -> Dict[str, Any]:
             "storage_account_key": connection_data["accountkey"],
         }
 
-    if "AZURE_APPLICATION_CREDENTIALS" in os.environ:
```

Comment:
Why do we remove this?

Reply:
oh my bad, I didn't notice I removed that